packages/monaco/src/browser/monaco-init.ts

```diff
@@ -23,26 +23,11 @@
  * might call `initialize()` while being constructed.
  * The service descriptors require a constructor function, so we declare dummy class for each Monaco service we override. But instead of returning an instance of the dummy class,
  * we fetch the implementation of the monaco service from the inversify container.
- * The inversify-constructed services must not call StandaloneServices.get() or StandaloneServices.initialize() from their constructors. Calling `get`()` in postConstruct mehtods
+ * The inversify-constructed services must not call StandaloneServices.get() or StandaloneServices.initialize() from their constructors. Calling `get`()` in postConstruct methods
  * is allowed.
  */
 
 // Before importing anything from monaco we need to override its localization function
-import * as MonacoNls from '@theia/monaco-editor-core/esm/vs/nls';
-import { nls } from '@theia/core/lib/common/nls';
-import { FormatType, Localization } from '@theia/core/lib/common/i18n/localization';
-
-Object.assign(MonacoNls, {
```

Comment:
Was there a specific reason for removing this? This breaks all localization of Monaco code.

Reply:
Please open an issue

Reply:
I also only noticed this because someone already opened https://github.com/eclipse-theia/theia/issues/14995.